starter-code/app.js

```diff
@@ -3,3 +3,40 @@ const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+app.set("views", __dirname + "/views");
+app.set("view engine", "ejs");
+
+app.get('/', (req, res) => {
+    res.render("index");
```

Comment:
Here, in fact, you want to display the same thing that in /random (not only the index but with a joke), for doing that:
```
res.redirect('/random');
```